client/my-sites/themes/get-theme-showcase-logged-out-seo-content.js

```diff
@@ -0,0 +1,263 @@
+			header: translate( 'Find the perfect theme for your about website.' ),
+			description: translate(
+				"Start your link in bio website with the perfect theme. Browse high-quality, professionally-designed options. Then, activate the one that's right for you."
+			),
+		},
+		free: {
+			title: translate( 'Free Link in Bio WordPress Themes' ),
+			header: translate( 'Discover the perfect free theme for your link in bio website.' ),
+			description: translate(
+				"Launch your link in bio website with the perfect free theme. Browse high-quality, professionally-designed options. Then, activate the one that's right for you."
+			),
+		},
+		premium: {
+			title: translate( 'Premium Link in Bio WordPress Themes' ),
+			header: translate( 'Choose premium themes for your link in bio website.' ),
+			description: translate(
+				"Enhance your link in bio website with the perfect premium theme. Available on all Premium, Business, and Ecommerce plans. Activate the one that's right for you."
+			),
+		},
+		marketplace: {
+			title: translate( 'Paid Link in Bio WordPress Themes' ),
+			header: translate( 'Select paid themes for your link in bio website.' ),
+			description: translate(
+				"Improve your link in bio website with the perfect paid theme. Browse high-quality, professionally-designed options. Then, purchase the one that's right for you."
+			),
+		},
+	},
+	newsletter: {
+		all: {
+			title: translate( 'Newsletter WordPress Themes' ),
+			header: translate( 'Find the perfect theme for your site-hosted newsletter.' ),
+			description: translate(
+				"Start your newsletter with the perfect theme. Browse high-quality, professionally-designed options. Then, activate the one that's right for you."
+			),
+		},
+		free: {
+			title: translate( 'Free Newsletter WordPress Themes' ),
+			header: translate( 'Discover the perfect free theme for your site-hosted newsletter.' ),
+			description: translate(
+				"Launch your newsletter with the perfect free theme. Browse high-quality, professionally-designed options. Then, activate the one that's right for you."
+			),
+		},
+		premium: {
+			title: translate( 'Premium Newsletter WordPress Themes' ),
+			header: translate( 'Choose premium themes for your site-hosted newsletter.' ),
+			description: translate(
+				"Enhance your newsletter with the perfect premium theme. Available on all Premium, Business, and Ecommerce plans. Activate the one that's right for you."
+			),
+		},
+		marketplace: {
+			title: translate( 'Paid Newsletter WordPress Themes' ),
+			header: translate( 'Select paid themes for your site-hosted newsletter.' ),
+			description: translate(
+				"Improve your newsletter with the perfect paid theme. Browse high-quality, professionally-designed options. Then, purchase the one that's right for you."
+			),
+		},
+	},
+};
+
+export default function getThemeShowcaseLoggedOutSeoContent( filter, tier ) {
+	const parsedFilter = filter?.replace( 'subject:', '' ) || 'recommended';
```

Comment:
This causes the title / header / description to suddenly change when filtering by feature.

See this example and note how the header changes from "Find the perfect theme for your blog." to "Find the perfect theme for your website." after selecting a feature:


https://github.com/Automattic/wp-calypso/assets/1233880/a7e78968-fda4-401e-a9ea-ba3379509347




I think it should remain as "Find the perfect theme for your blog." since the Blog category is still selected.

Reply:
Excellent catch @mmtr! Should be fixed with f2c6f3f.